File: gluecodium/src/test/resources/smoke/field_constructors/output/android/com/example/smoke/MutableStructNoClash.java
/*
 *
 */
package com.example.smoke;
import android.support.annotation.NonNull;
public final class MutableStructNoClash {
    @NonNull
    public String stringField;
    public int intField;
    public boolean boolField;
    public MutableStructNoClash() {
        this.stringField = "nonsense";
        this.intField = 42;
        this.boolField = true;
    }
}

File: gluecodium/src/test/resources/smoke/field_constructors/output/android-kotlin/com/example/smoke/MutableStructNoClash.kt
/*

 *
 */

@file:JvmName("MutableStructNoClashExtensions")

package com.example.smoke


class MutableStructNoClash {
    @JvmField var stringField: String
    @JvmField var intField: Int
    @JvmField var boolField: Boolean





    constructor() {
        this.stringField = "nonsense"
        this.intField = 42
        this.boolField = true
    }





}

